processing/src/main/java/org/apache/druid/query/rowsandcols/semantic/DefaultFramedOnHeapAggregatable.java

```diff
@@ -321,7 +321,7 @@ public static Object cloneAggValue(AggregatorFactory aggFactory, Object value)
     Aggregator combiningAgg = aggFactory.getCombiningFactory().factorize(combiningFactory);
 
     combiningAgg.aggregate();
-    return combiningAgg.get();
```

Comment:
note: I think we miss `combiningAgg.close()` here - not sure if hat matters...but could be fixed with a try-with-resources

Reply:
added it